pkg/trace/api/api.go

```diff
@@ -516,13 +515,20 @@ func (r *HTTPReceiver) handleTraces(v Version, w http.ResponseWriter, req *http.
 		atomic.AddInt64(&ts.PayloadRefused, 1)
 		return
 	}
+	if err == errInvalidHeaderTraceCountValue {
```

Comment:
As I explained below, an error here should not count as `success:false`. Perhaps you can scope the error to the `if` condition and define a new one below.